modules/configuring-huge-pages.adoc

```diff
@@ -1,7 +1,7 @@
 [id="configuring-huge-pages-{context}"]
 = Configuring huge pages
 
-Nodes must pre-allocate huge pages used in a {product-title} cluster. Use the
+Nodes must pre-allocate huge pages used in an {product-title} cluster. Use the
```

Comment:
@mtnbikenc qq if you don't mind: how can the Tuning operator allocate the huge pages if the node is not even part of the cluster ?  or am i reading it wrong ?

Reply:
This appears to be a doc module update that is not related to byo rhel scaleup.  Details for huge pages are covered under [Scalability and performance](https://docs.openshift.com/container-platform/4.1/scalability_and_performance/what-huge-pages-do-and-how-they-are-consumed-by-apps.html).